### PR TITLE
bump version to 15.0.1

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

**Review by rebeccahhh — APPROVED** [review]
:grinning:

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+- awx.awx collection modules now provide a clearer error message for incompatible versions of awxkit - https://github.com/ansible/awx/issues/8127
+- Fixed a bug in notification bodies that contain certain unicode characters - https://github.com/ansible/awx/issues/7400
+- Fixed a bug that prevents the deletion of Workflow Approval records - https://github.com/ansible/awx/issues/8305
+- Fixed a bug that broke the selection of webhook credentials - https://github.com/ansible/awx/issues/7892
+- Fixed a bug which can cause confusing behavior for social auth logins across distinct browser tabs - https://github.com/ansible/awx/issues/8154
+- Fixed a bug in the output of Workflow Job Templates using the `awx export` tool - https://github.com/ansible/awx/issues/7798
+- Fixed a race condition that can lead to missing hosts when running parallel inventory syncs - https://github.com/ansible/awx/issues/5571
+- Fixed an HTTP 500 error when certain LDAP group parameters aren't properly set - https://github.com/ansible/awx/issues/7622
+- Updated a few dependencies in response to several CVEs:
+    . CVE-2020-7720
```
Comment (shanemcd) [inline CHANGELOG.md]: This list renders weirdly in the markdown. View with:

image

Looks like:

image

Comment (ryanpetrello) [reply CHANGELOG.md]: Fixed!

**Comment by jbradberry** [comment]
Does #7847 rate?

**Comment by ryanpetrello** [comment]
@jbradberry fixed!

**Review by jbradberry — APPROVED** [review]

**Review by jakemcdermott — APPROVED** [review]

**Comment by ryanpetrello** [comment]
regate

**Review by beeankha — APPROVED** [review]

**Pull request merged**